### ISSUE TITLE
change directory for mocks

### ISSUE THREAD

**Issue by upvince** [maitria/mock-api-server#24]
Is there a way to change the standard folder `/test/mock-api` to something else ?

**Comment by eraserhd** [comment]
I don't believe so.  I'm no longer using this myself.  Contributions are welcome.
